core/src/main/java/org/mule/runtime/core/internal/policy/PolicyNextActionMessageProcessor.java

```diff
@@ -27,24 +24,20 @@
 import org.mule.runtime.core.api.MuleContext;
 import org.mule.runtime.core.api.context.notification.FlowStackElement;
 import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.api.policy.PolicyNotificationHelper;
 import org.mule.runtime.core.api.processor.Processor;
 import org.mule.runtime.core.internal.context.notification.DefaultFlowCallStack;
 import org.mule.runtime.core.internal.exception.MessagingException;
 import org.mule.runtime.core.internal.message.InternalEvent;
-import org.mule.runtime.core.internal.message.InternalEvent.Builder;
-import org.mule.runtime.core.privileged.event.BaseEventContext;
-import org.mule.runtime.core.privileged.event.PrivilegedEvent;
-
-import org.reactivestreams.Publisher;
-import org.slf4j.Logger;
-
-import com.google.common.collect.ImmutableSet;
 
 import java.util.Set;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Set'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7760&issues=AWnHtdTcotIXnmsUhQF_&open=AWnHtdTcotIXnmsUhQF_)

[ISSUE_KEY]: AWnHtdTcotIXnmsUhQF_